apiserver/common/package_test.go

```diff
@@ -1,18 +1,21 @@
 // Copyright 2021 Canonical Ltd.
 // Licensed under the AGPLv3, see LICENCE file for details.
 
-package common
+package common_test
```

Comment:
Can you make a note, that all these mock files should be in the common package. I can live with it atm, but we should sweep around to fix this.